### ISSUE TITLE
Export to a format suitable for a relational database

### ISSUE THREAD

**Issue by mvglasow** [mitre/cti#144]
Is there a way to export the framework to a format suitable for import into a relational database (e.g. CSV)? I am particularly interested in:

* Frameworks
* Tactics
* Techniques
* Sub-techniques

each with their ID, name and description, as well as the relationships between these entities (which tactics are part of which framework, which tactics are related to which techniques).

If there is a way to do this, documentation in a prominent place would be appreciated. (Am I really the first to encounter this issue?) If not, then this would be a feature request.

**Comment by isaisabel** [comment]
We're in the process of building a STIX to Excel converter which may suit your needs. You can find the WIP code for the converter here: https://github.com/mitre-attack/attack-scripts/tree/feature/%2345-attack-to-excel/attackToExcel

There is another project announced at the recent ATT&CKcon Power Hour (the ATT&CK Workbench project) which will provide a way to ingest the data into a Mongodb database and an official implementation of such a database and relevant APIs. That'll be coming out sometime this year.

**Comment by mvglasow** [comment]
Thanks, I just gave it a try. Getting it to run took me a while—had to fire up the Linux VM after all attempts with Cygwin failed.

For anyone trying the same, the requirements from the README in the root dir also apply to this module. Apart from python3, you might also need python3-wheel, python3-venv, pythin3-dev, gcc and gcc-c++.

Looks great, except that the tactics to techniques mapping is hard to parse. Adding these mappings e.g. on the relationship tab would help a lot.

**Comment by isaisabel** [comment]
Thanks for the feedback! Technically tactics and techniques are not related via relationships, but rather via `kill_chain_phases` on the technique objects, which is why they aren't represented like the other relationship types (e.g procedure examples). 

That said, we definitely should improve the tactic/technique mappings in a clearer format, I do understand why that would be difficult to work with in their current form.
